utilities.py

```diff
@@ -436,3 +432,20 @@ def print_group_path_for_DANSE(path):
 
 def print_error(error, position):
     print(defs.Messages.ERROR_IN.format(position = position, type_error_name = type(error).__name__, error = error), flush=True)
+
+#*************************************************************** OTHER FUNCTIONS ********************************************
+def def_chunk_size(data):
```

Comment:
Maybe, use directly the shape? instead of sending the whole data

Reply:
I found it nice to do `f.create_dataset(..., data=time_, chunks=def_chunk_size(time_), ...)` 
But indeed maybe I could just send the shape

Reply:
Yes, keep it like that, but instead of `def_chunk_size(time_)`, do `def_chunk_size(time_.shape)`